<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>49</y>
      <width>741</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="orderGrid">
     <item row="0" column="0">
      <widget class="QLabel" name="orderLabel">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>741</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridPart">
     <item row="0" column="0">
      <widget class="QLabel" name="partLabel">
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="staticOrderLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Order</string>
    </property>
   </widget>
   <widget class="QLabel" name="staticPartLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Parts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goodButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>771</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GOOD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="errorButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>450</y>
      <width>371</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NOT GOOD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="customErrorButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>771</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CUSTOM ERROR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ssButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>771</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>STOP EVERYTHING</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="errorLine">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>560</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="goodLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Good</string>
    </property>
   </widget>
   <widget class="QLabel" name="badLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bad</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Elapsed time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="elapsedTime">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>5</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="goodBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="badBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>470</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
